docs/develop/06_contribute.md

```diff
@@ -0,0 +1,102 @@
+
+## Treasury Proposals
+
+A Treasury proposal is a request to receive funds from the Treasury pool. 
+The proposal should begin with a clear title, a written outline of the idea, and a discussion about implementation or deliverables as outlined above for feature requests. 
+The proposal should be for something that changes or adds value to KILT in a meaningful way. 
+In general, a Treasury proposal spent occurs after completing all outlined deliverables and not before.
+Thus, it is recommended to rather open multiple subsequent milestone-based proposals instead of a large one to fund contributions.
+
+In addition you should:
+
+- Explain any deliverables that have already been achieved
+- Outline what needs to be done for the proposal to be completed
+
+The proposal should then be discussed with  the community (including, for example, the KILT Technical Committee, governance, or relevant project team) using community channels such as [Discord](https://discord.gg/7uyfMXh6AT) or [Polkassembly](https://kilt.polkassembly.network/discussions).
+If the community is not in agreement with the proposal, it is unlikely that it would be approved by governance.
+See the [guide to creating a proposal](../participate/03_treasury_proposal.md). 
+
+## Tips
+
+A more agile and lightweight process to receive rewards for contributing to the KILT Protocol are tips.
+Even though the funds also come from the Treasury, the procedure is more hands-down.
+The major difference compared to Treasury proposals is that for Tips, determining the bounty amount is part of the course of tipping.
+In other words, the final tip amount is not clear beforehand and the group of pre-determined stakeholders comes to consensus on how much should be paid.
+Eventually, the median of proposed tips will be awarded from the Treasury.
+Currently, the tippers include all Council members and other core code contributors.
+
+Both operations are similar in the sense that there must be someone (called _Finder_) to open the tipping process by providing a reason in form of a URL or an explanation on [Polkassembly](https://kilt.polkassembly.network/).
+Albeit, tips do not require an extensive document for that, e.g., a URL to the pull request or the blog post suffices.
+In case the Finder is part of the group which decides about the bounty award, no deposit needs to be made.
+Moreover, the beneficiary will receive the entire tip without any reductions.
+Otherwise, a small deposit, which is dependant on the length of the provided reason, is required to be reserved.
+It will be made available after the tipping process has finished.
+Additionally, the Finder also receives a minor Finder's fee of 20% which is subtracted from the final tip amount.
+
+Therefore, even if you are not a contributor, you can open a tipping process for someone else and receive a smaller portion of their potential reward.
```

Comment:
I think this sentence is quite important and deserves to be highlighted in some way.